tox.ini

```diff
@@ -1,10 +1,23 @@
 [tox]
-envlist = py26, py27, py32, py33, py34, pypy
+envlist = py26, py27, py32, py33, py34, pypy, gae
 
 [testenv]
+passenv = PYTHONPATH TRAVIS*
```

Comment:
Do we need the `TRAVIS*`?
How about setting a env var `GAE_PYTHONPATH` and then do `setenv = PYTHONPATH={env:GAE_PYTHONPATH}`?
This way we should not pick up anything which our users have set.
(or `URLLIB3_GAE_PYTHONPATH`)